src/tools/illink/test/Mono.Linker.Tests.Cases/Statics/UnusedStaticFieldInitializer.cs

```diff
@@ -18,5 +20,16 @@ public static void Foo ()
 			{
 			}
 		}
+
+		class C2
+		{
+			public static object o = new object ();
```

Comment:
If it's not beforefieldinit, the cctor will be kept because we need it for the ctor. Would it test anything interesting?

(We do have a test for beforefieldinit/not-beforefieldinit cctors).